binary.go

```diff
@@ -0,0 +1,45 @@
+package cleisthenes
+
+import "sync"
+
+type Binary = bool
+
+const (
+	One  Binary = true
+	Zero        = false
+)
+
+type BinaryState struct {
+	sync.RWMutex
+	value     Binary
+	undefined bool
```

Comment:
How about 
```go
type Undefined = bool
```
It seems to be more meaningful.

Reply:
I think it is too trivial. and just bool still straightforward